Zend/tests/024.phpt

```diff
@@ -41,17 +41,17 @@ NULL
 Warning: Undefined variable $b in %s on line %d
 int(1)
 
-Warning: Trying to get property '1' of non-object in %s on line %d
+Warning: Access to property '1' on int in %s on line %d
```

Comment:
If it's knowingly a property read, would it make sense to keep the "get" wording as opposed to "access" which may also mean "write"? If there's more than one get/set statements on the same line (may be the case for autogenerated code), it might be useful to have an extra pointer on which exactly of them (a "get" or "set") is causing the trouble.

Reply:
Yeah, I'm also not convinced that this message is better than the previous one (apart from making "non-object" more specific).

Reply:
So you'd be happy with the `Trying to get ...`, right? I'll change it then. It felt a bit awkward for me, but I can live with it, since it's more specific for sure.

Reply:
Or `Trying to read` maybe. But something that includes get/read.

Reply:
The other message is "Attempt to assign property xxx" so maybe "Attempt to read property xxx" is also a choice.

Reply:
`Read` definitely sounds more natural for me, so I'll use this one.

Reply:
> The other message is "Attempt to assign property xxx" so maybe "Attempt to read property xxx" is also a choice.

Oh, why I didn't check the format used by the writing side... That said, `Attempt to read property ...` sounds like the best choice!